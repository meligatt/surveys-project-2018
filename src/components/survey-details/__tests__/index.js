import React from 'react';
import SurveyDetails from '../index';

describe('SurveyDetails', () => {
  it('renders correctly', () => {
    const wrapper = shallow(
      <SurveyDetails />
    );

    expect(wrapper).toMatchSnapshot();
  });
});
