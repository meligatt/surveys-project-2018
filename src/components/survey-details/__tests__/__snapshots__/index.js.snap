// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SurveyDetails renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <SurveyDetails />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        undefined,
        <ul
          className="survey-details__list"
>
          <li
                    className="survey-details__item"
          >
                    Participant count: 
          </li>
          <li
                    className="survey-details__item"
          >
                    Response rate: 
                    %
          </li>
          <li
                    className="survey-details__item"
          >
                    Submitted response count: 
          </li>
</ul>,
      ],
      "className": "survey-details",
    },
    "ref": null,
    "rendered": Array [
      undefined,
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <li
              className="survey-details__item"
>
              Participant count: 
</li>,
            <li
              className="survey-details__item"
>
              Response rate: 
              %
</li>,
            <li
              className="survey-details__item"
>
              Submitted response count: 
</li>,
          ],
          "className": "survey-details__list",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "Participant count: ",
                undefined,
              ],
              "className": "survey-details__item",
            },
            "ref": null,
            "rendered": Array [
              "Participant count: ",
              undefined,
            ],
            "type": "li",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "Response rate: ",
                undefined,
                "%",
              ],
              "className": "survey-details__item",
            },
            "ref": null,
            "rendered": Array [
              "Response rate: ",
              undefined,
              "%",
            ],
            "type": "li",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "Submitted response count: ",
                undefined,
              ],
              "className": "survey-details__item",
            },
            "ref": null,
            "rendered": Array [
              "Submitted response count: ",
              undefined,
            ],
            "type": "li",
          },
        ],
        "type": "ul",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          undefined,
          <ul
            className="survey-details__list"
>
            <li
                        className="survey-details__item"
            >
                        Participant count: 
            </li>
            <li
                        className="survey-details__item"
            >
                        Response rate: 
                        %
            </li>
            <li
                        className="survey-details__item"
            >
                        Submitted response count: 
            </li>
</ul>,
        ],
        "className": "survey-details",
      },
      "ref": null,
      "rendered": Array [
        undefined,
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <li
                className="survey-details__item"
>
                Participant count: 
</li>,
              <li
                className="survey-details__item"
>
                Response rate: 
                %
</li>,
              <li
                className="survey-details__item"
>
                Submitted response count: 
</li>,
            ],
            "className": "survey-details__list",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  "Participant count: ",
                  undefined,
                ],
                "className": "survey-details__item",
              },
              "ref": null,
              "rendered": Array [
                "Participant count: ",
                undefined,
              ],
              "type": "li",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  "Response rate: ",
                  undefined,
                  "%",
                ],
                "className": "survey-details__item",
              },
              "ref": null,
              "rendered": Array [
                "Response rate: ",
                undefined,
                "%",
              ],
              "type": "li",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  "Submitted response count: ",
                  undefined,
                ],
                "className": "survey-details__item",
              },
              "ref": null,
              "rendered": Array [
                "Submitted response count: ",
                undefined,
              ],
              "type": "li",
            },
          ],
          "type": "ul",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
